client/components/tinymce/plugins/media/image-orientation-utils.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Please keep in mind [coding conventions for JavaScript](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md) in this new file.

```
eslint client/components/tinymce/plugins/media/image-orientation-utils.js 

/Users/andrew/Documents/Code/wp-calypso-krisk/client/components/tinymce/plugins/media/image-orientation-utils.js
  14:3   warning  Expected indentation of 1 tab character but found 0  indent
  14:14  warning  Unexpected space between function name and paren     no-spaced-func
  15:5   warning  Expected indentation of 1 tab character but found 0  indent
  16:7   warning  Expected indentation of 1 tab character but found 0  indent
  21:7   warning  Expected indentation of 1 tab character but found 0  indent
  22:7   warning  Expected indentation of 1 tab character but found 0  indent
  23:7   warning  Expected indentation of 1 tab character but found 0  indent
  26:2   warning  Unnecessary semicolon                                no-extra-semi
  39:3   warning  Expected indentation of 1 tab character but found 0  indent
  40:5   warning  Expected indentation of 1 tab character but found 0  indent
  43:11  warning  Expected indentation of 1 tab character but found 0  indent
  44:13  warning  Expected indentation of 1 tab character but found 0  indent
  46:13  warning  Expected indentation of 1 tab character but found 0  indent
  52:2   warning  Unnecessary semicolon                                no-extra-semi

✖ 14 problems (0 errors, 14 warnings)
```

Depending on your editor, I'd recommend installing both an [EditorConfig](http://editorconfig.org/) and [ESLint](http://eslint.org/docs/user-guide/integrations) plugin to make these issues more obvious, and to not interfere with your personal editor settings outside of Calypso.